mindjunkies/courses/models.py

```
@@ -155,3 +155,23 @@ class Meta:
+
+
+
+
+
+class CourseToken(models.Model):
+    STATUS_CHOICES = [
+        ('pending', 'Pending'),
+        ('accepted', 'Accepted'),
+        ('running', 'Running'),
+    ]
+    
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)